### ISSUE TITLE
While searching topic in channel list, it shows error.

### ISSUE THREAD

**Issue by swapnil106111** [learningequality/kolibri#2311]
## Summary
When learner search for a topic in search option, it gives error.
My server contains channel 8fc515793aad4e52b490b03c48ad21de (Maharashtra State English Medium). 
Note: It is not channel specific.It is coming across the channels.

## System information

 - Version: Kolibri 0.4.5
 - Operating system: Ubuntu 14.04
 - Browser: Chrome

## How to reproduce

1. Login with learner.
2. Search for a topic in search button besides the channel list.

## Screenshots
![2](https://user-images.githubusercontent.com/12776071/31003738-d4a27c38-a50e-11e7-8cab-647a4b4788d0.png)

![1](https://user-images.githubusercontent.com/12776071/31003747-dcf0acac-a50e-11e7-87d7-eb819e22eef5.png)

**Comment by christianmemije** [comment]
I get the following. Not sure if it is the same error.

```
AttributeError at /api/content/8fc515793aad4e52b490b03c48ad21de/contentnode/
'list' object has no attribute 'prefetch_related'
```

**Comment by rtibbles** [comment]
Any fuller traceback? Guessing that maybe our search filter is returning a list rather than a queryset?

**Comment by christianmemije** [comment]
```
Environment:


Request Method: GET
Request URL: http://127.0.0.1:8000/api/content/6a58e736c40f427ca7ef71fb25b4f067/contentnode/?page=1&page_size=20&search=math&1506713276617=1506713276617

Django Version: 1.9.7
Python Version: 2.7.12
Installed Applications:
[u'kolibri.core',
 u'django.contrib.admin',
 u'django.contrib.auth',
 u'django.contrib.contenttypes',
 u'django.contrib.sessions',
 u'django.contrib.messages',
 u'django.contrib.staticfiles',
 u'kolibri.auth.apps.KolibriAuthConfig',
 u'kolibri.content',
 u'kolibri.logger',
 u'kolibri.tasks.apps.KolibriTasksConfig',
 u'django_q',
 u'kolibri.core.webpack',
 u'kolibri.core.exams',
 u'kolibri.core.discovery',
 u'rest_framework',
 u'django_js_reverse',
 u'jsonfield',
 u'morango',
 u'kolibri.plugins.management',
 u'kolibri.plugins.learn',
 u'kolibri.plugins.document_pdf_render',
 u'kolibri.plugins.html5_app_renderer',
 u'kolibri.plugins.video_mp4_render',
 u'kolibri.plugins.audio_mp3_render',
 u'kolibri.plugins.setup_wizard',
 u'kolibri.plugins.coach',
 u'kolibri.plugins.user',
 u'kolibri_exercise_perseus_plugin']
Installed Middleware:
(u'django.contrib.sessions.middleware.SessionMiddleware',
 u'django.middleware.common.CommonMiddleware',
 u'django.middleware.csrf.CsrfViewMiddleware',
 u'kolibri.plugins.setup_wizard.middleware.SetupWizardMiddleware',
 u'kolibri.auth.middleware.CustomAuthenticationMiddleware',
 u'kolibri.content.middleware.ContentDBRoutingMiddleware',
 u'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
 u'django.contrib.messages.middleware.MessageMiddleware',
 u'django.middleware.clickjacking.XFrameOptionsMiddleware',
 u'django.middleware.security.SecurityMiddleware')



Traceback:

File "/home/christian/.virtualenvs/kolibri/local/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  149.                     response = self.process_exception_by_middleware(e, request)

File "/home/christian/.virtualenvs/kolibri/local/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  147.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/home/christian/.virtualenvs/kolibri/local/lib/python2.7/site-packages/django/views/decorators/csrf.py" in wrapped_view
  58.         return view_func(*args, **kwargs)

File "/home/christian/.virtualenvs/kolibri/local/lib/python2.7/site-packages/rest_framework/viewsets.py" in view
  87.             return self.dispatch(request, *args, **kwargs)

File "/home/christian/.virtualenvs/kolibri/local/lib/python2.7/site-packages/rest_framework/views.py" in dispatch
  466.             response = self.handle_exception(exc)

File "/home/christian/.virtualenvs/kolibri/local/lib/python2.7/site-packages/rest_framework/views.py" in dispatch
  463.             response = handler(request, *args, **kwargs)

File "/home/christian/.virtualenvs/kolibri/local/lib/python2.7/site-packages/rest_framework/mixins.py" in list
  45.             return self.get_paginated_response(serializer.data)

File "/home/christian/.virtualenvs/kolibri/local/lib/python2.7/site-packages/rest_framework/serializers.py" in data
  674.         ret = super(ListSerializer, self).data

File "/home/christian/.virtualenvs/kolibri/local/lib/python2.7/site-packages/rest_framework/serializers.py" in data
  239.                 self._data = self.to_representation(self.instance)

File "kolibri/content/serializers.py" in to_representation
  85.         data = data.prefetch_related(

Exception Type: AttributeError at /api/content/6a58e736c40f427ca7ef71fb25b4f067/contentnode/
Exception Value: 'list' object has no attribute 'prefetch_related'

```

**Comment by rtibbles** [comment]
Yeah, that seems to be the issue - does the same thing happen on 0.5?

**Comment by christianmemije** [comment]
Issue is not in 0.5

**Comment by rtibbles** [comment]
Can you try cherry picking this commit onto 0.4 and see if it fixes the issue/doesn't break anything else? https://github.com/learningequality/kolibri/commit/e0887a08f3ee3ad2113537aa62dbc72eaab96950

**Comment by christianmemije** [comment]
Applied commit. Fixes issue. Does not seem to cause regressions.

**Comment by rtibbles** [comment]
Excellent!

**Comment by christianmemije** [comment]
Fixed in #2315

**Comment by swapnil106111** [comment]
@rtibbles  Tested on Kolibri0.4.6beta1. Working fine.

![2311_1](https://user-images.githubusercontent.com/12776071/31373614-5edfb544-adb8-11e7-9b95-bb589a56bdcd.png)
![2311_2](https://user-images.githubusercontent.com/12776071/31373615-5f4613d4-adb8-11e7-9b20-f4521c725f6f.png)
